mail/rspamd/Portfile

```diff
@@ -2,78 +2,98 @@
 
 PortSystem          1.0
 PortGroup           cmake 1.0
-PortGroup           github 1.0
 
-github.setup        vstakhov rspamd 0.8.3
```

Comment:
Why are you removing the github portgroup? You are duplicating it's functionality with added line below.

Reply:
Probably because I didn't rtfm. I see from https://guide.macports.org/chunked/reference.portgroup.html that this should be something like

```
github.setup        rspamd rspamd 1.9.2
```

Then delete some of the other stuff. Is this correct?

Reply:
Yes, that is basically correct.